### PR TITLE
Flamalyzer git workflow

### PR DESCRIPTION
**Pull request by domeej**
Will run the Flamalyzer in the pipeline

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -54,4 +54,9 @@ jobs:
       - name: Go Generate
         run: |
           go generate ./...
-          git diff --quiet || (echo 'generated go files are not up to date, check go generate, go.sum and go.mod' ; git diff ; exit 1)
\ No newline at end of file
+          git diff --quiet || (echo 'generated go files are not up to date, check go generate, go.sum and go.mod' ; git diff ; exit 1)
+      - name: get Flamalyzer
```
Comment (tessig) [inline .github/workflows/main.yml:58]: I don't think we need to get it first (see above on misspell and ineffassign)

**Review by tessig — CHANGES_REQUESTED** [review]
We must definitely merge the cleanup branch first :)

File: .github/workflows/main.yml (hunk 1 above)
Comment (domeej) [reply .github/workflows/main.yml:58]: I thought the same but I get an error if I remove it 
```
no required module provides package flamingo.me/flamalyzer; to add it: go get flamingo.me/flamalyzer
```

Comment (domeej) [reply .github/workflows/main.yml:58]: another idea?

**Comment by tessig** [comment]
Closing because stale

**Pull request closed without merging**